modules/installation-uninstall-vsphere.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/uninstalling-cluster-vsphere.adoc
+
+[id="installation-uninstall-vsphere_{context}"]
+= Removing a cluster from vSphere
+
+You can remove a cluster that you installed on vSphere.
+
+.Procedure
```

Comment:
Any storage created by kubernetes should also be deleted. The storage looks to be created under the `kubevols` folder in the configured datastore. But it does not appear to be associated in any way to a particular cluster. I will look into this further to see if I can determine better guidance.